### ISSUE TITLE
qdb swallows (some) program output

### ISSUE THREAD

**Issue by thisguycodes** [quantopian/qdb#20]
I noticed when running the example that the `print` statements don't actually come out on stdout anymore, but instead go to the tmp file. With some testing I determined this is all print statements, but not writes directly to `sys.stdout`.

I don't necessarily disagree with having program output written to the tmp file (having the program output and the debugger output in the same place is convenient, and even mimics was the standard `pdb` does), nor am I advocating for use of print statements in code, but I definitely don't think that it should be modifying program behaviour like this, and this may cause qdb to be unusable in some situations. I haven't investigated how this gagging is done so I also don't know if it could affect things.

TLDR;
Code run with `set_trace()` should produce the same output as code run normally.
